src/main/scala/com/databricks/labs/overwatch/utils/Tools.scala

```diff
@@ -603,8 +604,7 @@ object Helpers extends SparkSessionWrapper {
    */
   def parListFiles(path: String): Array[String] = {
     try {
-      val conf = new Configuration()
-      val fs = new Path(path).getFileSystem(conf)
+      val fs = new Path(path).getFileSystem(spark.sparkContext.hadoopConfiguration)
```

Comment:
I have struggled with Serialization issues in the past. Is there a specific reason you moved this other than solely code cleanliness? Note the usage inside the lamba [HERE](https://github.com/databrickslabs/overwatch/blob/993ea86e956e7e01453fc58de33e2543392ea957/src/main/scala/com/databricks/labs/overwatch/pipeline/BronzeTransforms.scala#L672).

Did you test this with sparkEvents module to ensure spark events could be loaded without throwing `task not serializable` errors?

Reply:
if you don't have time to test, I will be testing it as part of the larger 0.4.2 release testing.

Reply:
yes, I tested with all modules, include `sparkEvents`